### ISSUE TITLE
Topic limit of charachters

### ISSUE THREAD

**Issue by AnneliesVerroken** [micado-eu/pa_application#141]
Is it possible to extend the amount of characters for a topic title?

Antwerpen is already 9 charachters ;)  and Dutch is clearly a language with some rather long words, especially if you try to use accessible language for non-native speakers ;)
I absolutely understand the need for short titles, but maybe up to 30 would be reasonable? 

Thank you!

**Comment by AnneliesVerroken** [comment]
The same goes for document titles: more characters please, our official documents seem to have some long titles

**Comment by gioppoluca** [comment]
it COULD be possible the problem is IF IT IS REASONABLE.
The problem is that in the smarthone the actual 25 character limit would generate a title on 3 lines that will look AWFUL.
Topics should be 1-2 words at maximum and since you are in Antwerpen there is no need to repeat it I imagine you try to write "Antwerpen's adminstrative procedure" or similar things, just remove Antwerpens's and you recover a LOT of characters

**Comment by AnneliesVerroken** [comment]
You are absolutely right, and I can find my way around it for the topics, but not for document names. An official name for a document is what is it, we can not put only half the name in the document title. 
The longest document name we have misses 6 charachters now.  It is not a joke, Dutch is a language that often uses long words, especially in administration.

**Comment by AnneliesVerroken** [comment]
Hi guys,

I am very sorry, but is there any chance the character limit for documents could be adapted quite soon? As said before, I miss 6 characters right now. Changing the document names is not as easy as it sounds, since it is the official name and everything has been sent to the translation office weeks ago. 
We will have tryouts for our user tests next week, so we would like to have all content published by then. 

Thank you!

**Comment by gioppoluca** [comment]
will be fixed in the upcoming release
